spokestack/wakeword/tflite.py

```diff
@@ -204,14 +163,8 @@ def _detect(self, context: SpeechContext) -> None:
     def reset(self) -> None:
         """ Resets the currect WakewordDetector state """
         self.sample_window.reset()
-        self.frame_window.reset().fill(0.0)
-        self.encode_window.reset().fill(0.0)
-        self.state = np.zeros(self.encode_model.input_details[1]["shape"], np.float32)
-        self.fft_window.reset()
+        self.frame_window.reset().fill(np.zeros((1, self.mel_width), np.float32))
```

Comment:
same note here as above, consider just passing the scalar 0 to fill the buffer, not shapes/dtypes needed